include/deal.II/matrix_free/matrix_free.h

```diff
@@ -640,6 +640,13 @@ class MatrixFree : public Subscriptor
    */
   //@{
   /**
+   * Returns whether a given FiniteElement @p fe is supported by this class.
+   */
+  template <int spacedim>
+  static
+  bool supports_fe (const FiniteElement<dim, spacedim> &fe);
```

Comment:
That's ok with me as well.

Your suggestion reads better in the context of a member function call, with `matrix_free_data.` prefixed. But it may look less obvious in the class documentation, where it simply says `supports(const FE &)` and you're back to asking yourself "who supports what here".

But both are fine with me.